contrib/docker/minimal-hardened-image/README.md

```diff
@@ -4,6 +4,19 @@ DockerHub images in general did not seem ideal for Backstage as the number of vu
 
 The `Dockerfile` in this directory uses a [wolfi-base](https://github.com/wolfi-dev) image from Chainguard Images. This improves the security of the application and reduces false positives in scanners.
 
+## Steps taken
```

Comment:
It's amazing to have this documented 👏🏻

Reply:
if more needs to be done let me know! marking as resolved